doc/design_AsyncNDRegisterAcessor_and_NumericAddressedBackend.dox

```diff
@@ -0,0 +1,60 @@
+namespace ChimeraTK {
+/**
+\page design_AsyncNDRegisterAccessor_NumericAddressed Design: AsyncNDRegisterAccessor and the asynchronous implelentation in the NumericAddressedBackend
```

Comment:
implelentation -> implementation

Reply:
The title refers to the numeric addressed backend, but it seems at least a great part of this document is independent it. I would remove the reference to the NA backend from the title.

Reply:
Well, the complete second section of the document is only about the NumericAddressedBackend. And this is the bigger part. Only the first section is generic.